### ISSUE TITLE
Add filter for time between TB/ALT measures

### ISSUE THREAD

**Issue by jwildfire** [SafetyGraphics/hep-explorer#23]
Idea here is to have a control that lets a user input the number of days and then limits the amount of time between the TB and ALT controls based on that user input.

Not exactly sure how to implement this though. Deriving a temporally “matched” set of ALT and TB is probably possible, but not trivial since we’d have to figure out some kind of bi-variate algorithm to define the maximal pair of values (instead of just doing a simple univariate max).

Maybe instead of the custom filter, maybe just provide details on demand. V0.2.0 includes some of thi with the marginal distribution (on mouseover) and 'snake plot' and summary table on click.  We could also consider encode time between measures visually by changing point opacity (or point size). 

Long story short, I'm leaning towards 'won't fix' for this, but need input from the group.
